### PR TITLE
update ParticleUniversePlugin header and add a python sample

### PR DESCRIPTION
**Pull request by niubaty**

### CODE REVIEW

**Comment by niubaty** [comment]
update ParticleUniversePlugin header and add a python sample

File: Samples/ParticleUniverseDemo/include/ParticleUniverseDemo.h [hunk 1]
```diff
@@ -15,6 +15,143 @@
 
 using namespace Ogre;
 
+
+/*
```
Comment (paroj) [inline Samples/ParticleUniverseDemo/include/ParticleUniverseDemo.h:20]: please remove commented out code

File: Samples/ParticleUniverseDemo/include/ParticleUniverseDemo.h [hunk 2]
```diff
@@ -52,11 +189,11 @@ class Sample_ParticleUniverseDemo : public OgreBites::ApplicationContext, public
 		Ogre::Viewport* viewport = getRenderWindow()->addViewport(mCamera);
 
 		// setup some basic lighting for our scene
-        mSceneMgr->setAmbientLight(ColourValue(0.3, 0.3, 0.3));
```
Comment (paroj) [inline Samples/ParticleUniverseDemo/include/ParticleUniverseDemo.h:55]: please keep code formatted with spaces

File: Samples/Py/OgreParticleuniverseLayer.cpp [hunk 3]
```diff
@@ -0,0 +1,47 @@
+#include "OgreParticleuniverseLayer.h"
+#include "Ogre.h"
+
+void OgrePartUniLayer::installPlugin(Ogre::Root * root) {
```
Comment (paroj) [inline Samples/Py/OgreParticleuniverseLayer.cpp:4]: in contrast to btOgre, this is not really needed. You can just wrap the relevant PU classes:
- ParticleSystemManager
- ParticleSystem

**Review by paroj — CHANGES_REQUESTED** [review]

File: Samples/Py/OgreParticleuniverseLayer.cpp (hunk 3 above)
Comment (niubaty) [reply Samples/Py/OgreParticleuniverseLayer.cpp:4]: I know. I just change directly in the repository. I should add the repository as my own submodule.
I would change them.

**Pull request closed without merging**